### PR TITLE
Azure Apps Overview updates

### LINKED ISSUE #13013: Clarify 64-bit scenario for Azure App Service
Hi Guys,

The whole x64 bits topic is confusing.
These days people want to run in x64 mode just because we have some load and need some memory to be available. And we want to understand how is that possible.

I beleive there are different languages, and even worse different deployment "modes". And, every language developer needs to understand what he must do to deploy his app as x64.

What's currently written is completely confusing me, as .NET dev and architect. Just try to read it and try ti understabnd what does it means:

"
Platform
Runtimes for 64-bit (x64) and 32-bit (x86) apps are present on Azure App Service. The .NET Core SDK available on App Service is 32-bit, but you can deploy 64-bit apps using the Kudu console or via MSDeploy with a Visual Studio publish profile or CLI command.
"

So, what is runtimes available, what is runtime? Is that OS? Is that .NET runtime? Is that .NET Core runtime or anything else?

&gt;&gt;The .NET Core SDK available on App Service is 32-bit
Seriously? Just a word befoew you sad that 64 bits runtime is available, so, why .NET Core SDK is not? And what is Kudu console? Needs to be clarified, e.g. we use Azure Pipelines

And then, what the hell Platform-&gt;64 Bit setting does

So, from the one who builds .NET Core and needs to deploy stuff as x64 the whole topic looks like a confusion.

Sorry for the a bit upset tone, and thatnk you for letting us post the feedback!

Linking:
https://github.com/aspnet/AspNetCore.Docs/issues/9800

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a2eab542-a420-d6c5-d314-a420837cf602
* Version Independent ID: 48c12168-54f3-0b83-f136-565f483a03e9
* Content: [Deploy ASP.NET Core apps to Azure App Service](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/azure-apps/?view=aspnetcore-2.2)
* Content Source: [aspnetcore/host-and-deploy/azure-apps/index.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/host-and-deploy/azure-apps/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-hostdeploy**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #13013
Addresses #12965

[Internal Review Topic](https://review.docs.microsoft.com/en-us/aspnet/core/host-and-deploy/azure-apps/index?view=aspnetcore-2.1&branch=pr-en-us-13322)

* Clarifications for 64-bit.
* Re-organize for preview vs. non-preview deployments:
  * The preview guidance in one section.
  * The general guidance on deployment in another (new) section.
* Explicitly say *where* the published app is to be deployed when dragging to the Kudu console. I'll finish the updates for #12965 on a PR to follow our merging of the Azure Apps and IIS troubleshooting topics. For now, this takes care of that issue on the Azure Apps Overview topic side of things.

### CODE REVIEW

File: aspnetcore/host-and-deploy/azure-apps/index.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Discover Azure App Service runtime execution limitations enforced by the Azure A
 
 ::: moniker range=">= aspnetcore-2.2"
 
-Runtimes for 64-bit (x64) and 32-bit (x86) apps are present on Azure App Service. The [.NET Core SDK](/dotnet/core/sdk) available on App Service is 32-bit, but you can deploy 64-bit apps using the [Kudu](https://github.com/projectkudu/kudu/wiki) console or via [MSDeploy with a Visual Studio publish profile or CLI command](xref:host-and-deploy/visual-studio-publish-profiles).
+Runtimes for 64-bit (x64) and 32-bit (x86) apps are present on Azure App Service. The [.NET Core SDK](/dotnet/core/sdk) available on App Service is 32-bit, but you can deploy 64-bit apps built locally using the [Kudu](https://github.com/projectkudu/kudu/wiki) console or the publish process in Visual Studio. For more information, see the [Publish and deploy the app](#publish-and-deploy-the-app) section.
```
Comment (bradygaster) [inline aspnetcore/host-and-deploy/azure-apps/index.md]: The past few days I was experimenting with a site we're porting to 3.0 preview 7. In so doing, we determined that the `.csproj` file had been set to 64b, and we kept getting ANCM errors on App Service. After a quick check we ascertained that the site was running in 32b. Once we changed it to 64b to match what was in the `.csproj` file it worked. I mention this as it bit us each time we set up a new slot. Somehow, seems we could encourage the customer to look at that setting if they see an ANCM issue. Might be good to show a screen shot in some doc, maybe not here, but somewhere.

Comment (guardrex) [reply aspnetcore/host-and-deploy/azure-apps/index.md]: `<Platform>` property?

The happy path AFAICT is not to set `<Platform>` (and we don't mention it anywhere AFAIK). I had a good outcome only using the switch for **Platform** in  **General settings**. It seems 🤞 to work well that way for an FDD.

It's a candidate for coverage in the troubleshooting topic ... I mean if we **don't** say anything about it here.

Comment (bradygaster) [reply aspnetcore/host-and-deploy/azure-apps/index.md]: good call!

Comment (guardrex) [reply aspnetcore/host-and-deploy/azure-apps/index.md]: I'll work it further on https://github.com/aspnet/AspNetCore.Docs/issues/12965.

Anything else on this PR?

Comment (bradygaster) [reply aspnetcore/host-and-deploy/azure-apps/index.md]: Nope all good.

**Pull request merged**